lib/constants/config.js

```diff
@@ -1,2 +1,7 @@
 export const CONFIG_LOAD = 'CONFIG_LOAD';
 export const CONFIG_RELOAD = 'CONFIG_RELOAD';
+export const CONFIG_ASYNC = 'CONFIG_ASYNC';
+
+export const BELL_STYLE = {
```

Comment:
I think this is not necessary. Or we should use it everywhere. Like here: https://github.com/zeit/hyper/pull/2938/files#diff-7da6d687f5a3beed5725bb8324e4b1adR26
or even here: https://github.com/zeit/hyper/pull/2938/files#diff-6d186b954a58d5bb740f73d84fe39073R32

But using 'SOUND' directly is ok imo.